src/net_processing.cpp

```diff
@@ -779,7 +781,10 @@ class PeerManagerImpl final : public PeerManager
     BanMan* const m_banman;
     ChainstateManager& m_chainman;
     CTxMemPool& m_mempool;
-    TxRequestTracker m_txrequest GUARDED_BY(::cs_main);
+
+    /** Protects tx download including TxRequest, rejection filter. */
+    mutable Mutex m_tx_download_mutex;
```

Comment:
Is the `mutable` keyword really required?

Reply:
Removed now, thanks